src/Processing/ImportProcessingService.php

```diff
@@ -113,8 +113,10 @@ public function __construct(QueueService $queueService, MappingConfigurationFact
         $this->configLoader = new ConfigurationPreparationService();
     }
 
-    public function processQueueItem(int $id)
+    public function processQueueItem(string $id)
```

Comment:
why did you change that?

Reply:
I would say we should do the casting outside if necessary ...

Reply:
I totally aggree with you. I will change that.